src/sentry/api_gateway/proxy.py

```diff
@@ -45,12 +45,15 @@ def proxy_request(request: Request, org_slug: str) -> StreamingHttpResponse:
     request_args = {
         "headers": request.headers,
         "params": dict(query_params) if query_params is not None else None,
-        "data": getattr(request, "body", None),
+        "files": getattr(request, "FILES", None),
+        "data": getattr(request, "body", None)
+        if request.content_type == "application/json"
```

Comment:
We might also have endpoints (especially in django views) that are taking form encoded POST bodies. Should we pass those through as well?

Reply:
If we don't then those pages will break right? I guess right now I'm not really seeing a reason why not to?

Reply:
Wouldn't the current logic not forward data for form-encoded requests? Request data is only forwarded for json requests.

Reply:
Updated.